### DISCUSSION TITLE
Supported MSSQL versions?

### DISCUSSION THREAD

**Discussion by TheWebbe** [serenity-is/Serenity#6399] [Help]
I might be missing it somewhere, but I couldn't seem to find a list of supported SQL versions. I know I had tried to connect to a SQL 2019 server a few months ago and it would not connect. Is there a list somewhere?

**Comment by wirble** [comment]
I have been using it on different versions of sql including 2019 and works fine.  It is your setup.
Did you go through the readme and installation guide or the docs?
Have you tried these:
https://serenity.is/docs/
https://github.com/serenity-is/Serenity
